### PR TITLE
Task 3 rectangular

### PR DESCRIPTION
**Pull request by Anameskhi**

### CODE REVIEW

File: task_3.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  def calculate
+    if is_rectangular? && is_isosceles?
+      'triangle is rectangle isosceles'
+    elsif is_rectangular?
+      'triangle is rectangle'
+    elsif is_equilateral?
+      'triangle is equilateral'
+    elsif is_isosceles?
+      'triangle is isosceles'
+
```
Comment (alexshvedko) [inline task_3.rb:15]: add `else`

Comment (Anameskhi) [reply task_3.rb:15]: okay

Comment (Anameskhi) [reply task_3.rb]: i added else parameter

File: task_3.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+class Triangle
```
Comment (alexshvedko) [inline task_3.rb:1]: Screenshot 2022-04-01 at 11 41 15

Comment (alexshvedko) [reply task_3.rb:1]: add validations

Comment (Anameskhi) [reply task_3.rb:1]: @alexshvedko  i added validatioin

File: task_3.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  def is_isosceles?
+    @side1 == @side2 || @side2 == @side3
+  end
+
+  def is_equilateral?
+    @side1 == @side2 && @side2 == @side3
+  end
+  def validate_sides(side)
+    if side.to_f == 0
+      puts "Eror!!! your selected side is string"
```
Comment (alexshvedko) [inline task_3.rb:36]: mistake

File: task_3.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      exit 1
+
```
Comment (alexshvedko) [inline task_3.rb:38]: remove empty line

File: task_3.rb [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    if is_rectangular? && is_isosceles?
+      'triangle is rectangle isosceles'
+    elsif is_rectangular?
+      'triangle is rectangle'
+    elsif is_equilateral?
+      'triangle is equilateral'
+    elsif is_isosceles?
+      'triangle is isosceles'
+    else
+      'your answers isn\'t triangle parameteres'
```
Comment (alexshvedko) [inline task_3.rb:19]: mistake

Comment (Anameskhi) [reply task_3.rb:25]: @alexshvedko i changed everything you told me . please check it

**Pull request merged**